integration-tests/integration/calendar-overtime.spec.js

```diff
@@ -15,50 +14,67 @@ context('A staff member can view their overtime calendar', () => {
     cy.task('stubNotificationCount')
     cy.login()
 
-    utilities.gotoPreviousCalendarDate('2020-03-01')
+    CalendarPage.verifyOnPage(moment().format('MMMM YYYY'))
+    cy.visit('/calendar/2020-04-01')
+    cy.get('[data-qa=previous]').click()
```

Comment:
gotoPreviousCalendarDate() was intended to get us from the current month view to a specified month which is so long ago it now takes ages. now simplified to just go to a month or 2 later then go back